src/SprykerMiddleware/Zed/Process/Business/Stream/CsvWriteStream.php

```diff
@@ -109,11 +105,11 @@ public function seek(int $offset, int $whence): int
     {
         $newPosition = $this->getNewPosition($offset, $whence);
         if ($newPosition < 0 || $newPosition > count($this->data)) {
-            return false;
+            return 0;
         }
         $this->position = $newPosition;
 
-        return true;
+        return 1;
```

Comment:
what does 1 mean?

Reply:
As for me, it should be *return boolean* here, but then I should update every stream + akeneo connector streams too, so I'm not sure I have enough time for it right now. It can be defined as the separate bug. For now, phpstan asks about int values.

Reply:
Then please make a class const as speaking one instead of magic number.

STATUS_SUCCESS or sth